Lesson05/js/ATM.js

```diff
@@ -0,0 +1,83 @@
+/*pseudocode area
+declare an object named ATM, it should have balance, options(functions), and a start functions
+
+declare 5 functions, getOption(), quit(), withdraw(), deposit(), and viewbalance()
+check negative balances and warn of low balance in withdraw()
+check deposit cap in deposit()
+
+use switch match 'N', 'W', 'D', 'Q' in start()
+*/
+
+let ATM = {
+  balance: 1000,
+  start: function() {
+    let shouldNotQuit = true;
+    while(shouldNotQuit) {
+      let option = (this.getOption()).toUpperCase();
+      switch (option) {
+        case 'Q':
+          this.quit();
+          shouldNotQuit = !shouldNotQuit;
```

Comment:
Since you're quitting, you could be specific and just say `shouldNotQuit = false`